doc/source/reference/c-api.iterator.rst

```diff
@@ -603,15 +602,18 @@ Construction and Destruction
             Indicates that only elements which the operand with
             the ARRAYMASK flag indicates are intended to be modified
             by the iteration. In general, the iterator does not enforce
-            this, it is up to the code doing the iteration to follow
-            that promise. Code can use the :c:func:`NpyMask_IsExposed`
-            inline function to test whether the mask at a particular
-            element allows writing.
+            this, it is up to the code doing the iteration to follow that
+            promise.
+
+            Essentially, this array is the mask for all WRITEMASKED operands.
+            Code uses the WRITEMASKED flag which indicates that only
+            elements where the chosen arraymask operand is True will
+            be written to.
 
             When this flag is used, and this operand is buffered, this
             changes how data is copied from the buffer into the array.
             A masked copying routine is used, which only copies the
-            elements in the buffer for which :c:func:`NpyMask_IsExposed`
+            elements in the buffer for which WRITEMASKED
```

Comment:
```
WRITEMASKED -> ``writemasked``
```
inside double-backticks and the first mention should have a link, something like
```
`writemasked <numpy.nditer>`
```
inside single-backticks so it becomes a link to [here](www.numpy.org/devdocs/reference/generated/numpy.nditer.html)